src/ListGroupItem.tsx

```diff
@@ -36,19 +36,21 @@ const propTypes = {
    */
   action: PropTypes.bool,
   /**
-   * Sets list item as active
+   * Sets list item as active.
    */
   active: PropTypes.bool,
 
   /**
-   * Sets list item state as disabled
+   * Sets list item state as disabled.
    */
   disabled: PropTypes.bool,
 
   eventKey: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
 
+  /** A callback function for when this component is clicked.  */
   onClick: PropTypes.func,
 
+  /** Providing a `href` will render the ListGroup.Item as an `<a>` element (unless `as` is provided). */
```

Comment:
The `action` prop is also required along with `href` to render an anchor.

Reply:
oh ok, thanks! I'll update it